homeassistant/helpers/event.py

```diff
@@ -403,7 +407,11 @@ def async_track_state_report_event(
     action: Callable[[Event[EventStateReportedData]], Any],
     job_type: HassJobType | None = None,
 ) -> CALLBACK_TYPE:
-    """Track EVENT_STATE_REPORTED by entity_id without lowercasing."""
+    """Track EVENT_STATE_REPORTED by entity_id without lowercasing.
```

Comment:
The "without lowercasing" is weird, there's no version which does lowercase the passed in entity ids.

Add it as a note instead:
```py
"""Track EVENT_STATE_REPORTED by entity_ids.

The passed in entity_ids will not be automatically lower cased.
```

And a corresponding note to the public function `async_track_state_change_event` which explains it lowercases.